examples/tests/qdec_callback/main.c

```diff
@@ -0,0 +1,48 @@
+#include <led.h>
+#include <qdec.h>
+#include <tock.h>
+#include <timer.h>
+
+static void qdec_callback(int qdec_num,
+                          __attribute__ ((unused)) int val,
+                          __attribute__ ((unused)) int arg2,
+                          __attribute__ ((unused)) void *ud) {
+  printf("QDEC Absolute Position: %d\n", qdec_num);
+
+  // Set LEDs based on acc value
+  // Note: LED rules arbitrarily chosen by developer
+  if (qdec_num > 0) {
+    led_on(0);
+    led_off(1);
+    led_off(2);
+  }  else if (qdec_num == 0) {
+    led_on(1);
+    led_off(0);
+    led_off(2);
+  } else if (qdec_num < 0) {
+    led_on(2);
+    led_off(0);
+    led_off(1);
```

Comment:
one potential problem with code like this is that not all Tock boards have at least 3 LEDs. not sure if that will be an issue upstream or not, but you can probably leave it for now

Reply:
Ah ok that makes sense-I can definitely remove this code if need be as its just a fun thing that exists rather than being critical at all to showing the user the QDEC works (the printf statements do that in much finer detail).